security-paseto/build.gradle

```diff
@@ -0,0 +1,37 @@
+ext {
+    jpasetoVersion = '0.7.0'
+    bouncyCastleVersion = '1.69'
+}
+
+dependencies {
+    api "io.micronaut:micronaut-http"
+    api "io.micronaut:micronaut-http-server"
+    api project(":security")
+    compileOnly "dev.paseto:jpaseto-api:$jpasetoVersion"
```

Comment:
we probably need this as `implementation` or `api`.

Reply:
@sdelamo I went with `api` since we might want to transitively export this dependency later.